input/SPT.yml

```diff
@@ -0,0 +1,13 @@
+mapper_class: 'MapperSPT'
+map_name: SPT
+file_map: /mnt/extraspace/damonge/Datasets/SPT/sptsz_planck_ymap_healpix/SPTSZ_Planck_min_variance_ymap.fits
+file_hm1: /mnt/extraspace/damonge/Datasets/SPT/sptsz_planck_ymap_healpix/SPTSZ_Planck_min_variance_ymap_half1.fits
+file_hm2: /mnt/extraspace/damonge/Datasets/SPT/sptsz_planck_ymap_healpix/SPTSZ_Planck_min_variance_ymap_half2.fits
+file_gp_mask: /mnt/extraspace/damonge/Datasets/SPT/sptsz_planck_ymap_healpix/SPTSZ_dust_mask_top_5percent.fits
+gp_mask_mode: 'default'
+file_sp_mask: /mnt/extraspace/damonge/Datasets/SPT/sptsz_planck_ymap_healpix/SPTSZ_point_source_mask_nside_8192_binary_mask.fits
+ps_mask_mode: ['default']
+coords: G
+custom_auto: True
+mask_name: mask_SPT
+path_rerun: '/mnt/extraspace/damonge/Datasets/SPT/xcell_runs'
```

Comment:
Should there be a `beam_info` section for this mapper?

Reply:
yes you are right! Added!